deploy/cdk/lib/user-content/index.ts

```diff
@@ -0,0 +1,119 @@
+import apigateway = require('@aws-cdk/aws-apigateway');
+import dynamodb = require('@aws-cdk/aws-dynamodb');
+import lambda = require('@aws-cdk/aws-lambda');
+import ssm = require('@aws-cdk/aws-ssm');
+import cdk = require('@aws-cdk/core');
+
+export interface UserContentStackProps extends cdk.StackProps {
+  readonly lambdaCodePath: string
+  readonly tokenAudiencePath: string
+  readonly tokenIssuerPath: string
+};
+
+export class UserContentStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props: UserContentStackProps) {
+    super(scope, id, props);
+
+    // Dynamo Tables
+    const userDynamoTable = new dynamodb.Table(this, 'UsersTable', {
+      billingMode: dynamodb.BillingMode.PAY_PER_REQUEST,
+      partitionKey: {
+        name: 'userName',
+        type: dynamodb.AttributeType.STRING
+      },
+      tableName: `${this.stackName}-users`,
+      removalPolicy: cdk.RemovalPolicy.DESTROY, // NOT recommended for production code
```

Comment:
Why does this say not recommended for production? Are there plans to change this?

Reply:
Do we know for certain that this will not destroy a table if it has data in it? Most resource types that have data will fail to be destroyed by CloudFormation when the removal policy is destroy (ex: s3, ecr, etc). I can't imagine Dynamo would be different, but I have not used it in CloudFormation yet, so if someone can confirm this. 

This will become especially important once we have a pipeline that continuously deploys this component. If someone accidentally removes this from the app, or if this resource gets refactored into a different stack in the app, the existing CloudFormation stack will attempt to destroy the existing table. Worst case, what we want is for this deployment execution to fail in test. Will still be a bit annoying, since we'll have to remediate that deploy with some manual steps, but at least the data will be protected.

Reply:
`DESTROY` will delete the table with data in it, hence the reason the `// NOT recommended for production code` comment. This was super useful when iterating on the stack, but can probably be removed now that this is "more stable".